lib/server.js

```diff
@@ -38,6 +34,18 @@ var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
 
 exports.create = function create(createOptions) {
 
+  // safely require the peer-dependencies
+  var React = util.safeRequire('react');
+  var Router = util.safeRequire('react-router');
+
+  if (!React) {
+    throw new Error('React is required as a peer dependency. Run `npm instal react`.')
+  }
+
+  if (!Router) {
+    throw new Error('React Router is required as a peer dependency. Run `npm instal react-router`.')
```

Comment:
absolutely, but you might want to check if the user is intending to use react-router or not before throwing here.

once simple way is to examine if `routes` property exists in creatOptions `createOptions.routes`